control-plane-operator/controllers/hostedcontrolplane/oauth/params.go

```diff
@@ -88,6 +89,38 @@ func NewOAuthServerParams(ctx context.Context, hcp *hyperv1.HostedControlPlane,
 			},
 		},
 	}
+	p.LivenessProbes = config.LivenessProbes{
+		oauthContainerMain().Name: {
+			Handler: corev1.Handler{
+				HTTPGet: &corev1.HTTPGetAction{
+					Scheme: corev1.URISchemeHTTPS,
+					Port:   intstr.FromInt(int(OAuthServerPort)),
+					Path:   "healthz",
+				},
+			},
```

Comment:
what makes this values being different than the ones for the kas?

```
			InitialDelaySeconds: 300,
			PeriodSeconds:       180,
			TimeoutSeconds:      160,
			FailureThreshold:    6,
			SuccessThreshold:    1,
```

Reply:
I believe this one is different and tailored to oauth